Marlin/src/inc/Conditionals_post.h

```diff
@@ -750,6 +750,12 @@
 // Sensors
 #define HAS_FILAMENT_WIDTH_SENSOR (PIN_EXISTS(FILWIDTH))
 #define HAS_FIL_RUNOUT (PIN_EXISTS(FIL_RUNOUT))
+// Multi Sensors
+#define HAS_FIL_RUNOUT0 (PIN_EXISTS(FIL_RUNOUT))
+#define HAS_FIL_RUNOUT1 (PIN_EXISTS(FIL_RUNOUT2))
+#define HAS_FIL_RUNOUT2 (PIN_EXISTS(FIL_RUNOUT3))
+#define HAS_FIL_RUNOUT3 (PIN_EXISTS(FIL_RUNOUT4))
+#define HAS_FIL_RUNOUT4 (PIN_EXISTS(FIL_RUNOUT5))
 
```

Comment:
My own ' essential and primitive ' sanity 
Good but , i'm sure some line are missing to avoid tons of people crying here